Source/Bundle+Convenience.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension Bundle {
+    class var resources: Bundle {
```

Comment:
Can you make this `static` for consistency with the rest of the project? Also, I'd like for this to be called `resourceBundle` as `resources` doesn't make it very clear that this a `Bundle` is being returned (as opposed to, say, an array of resources).